src/code/dialogs/blockersList.js

```diff
@@ -31,8 +31,8 @@ const BlockerList = WDialog.extend({
 
   removeHooks: function () {
     WDialog.prototype.removeHooks.call(this);
-    window.map.off("wasabee:uiupdate:mapdata", this.blockerlistUpdate, this);
-    window.map.off("wasabee:crosslinks:done", this.blockerlistUpdate, this);
+    window.map.off("wasabee:op:select wasabee:op:change", this.update, this);
```

Comment:
does this work? space separated list?

Reply:
yes: https://leafletjs.com/reference-1.7.1.html#evented-on
> You can also pass several space-separated types (e.g. 'click dblclick').